OpenUpSA/wazimap-ng-ui
# changing filename for chart download

## Description
Changed the file name so it contains the current filtering of the chart.

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/149

## How to test it locally
* Run the project
* Filter a chart
* Export as csv using the hamburger menu
* Check if the file name contains the filter 

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/100282402-922ffa00-2f7c-11eb-865a-a3e4647cb1f3.png)


## Changelog

### Added

### Updated
* `setChartMenu()` function so the title parameter contains the filter too

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
